pkg/packagekit/updates.jsx

```diff
@@ -471,6 +532,42 @@ class OsUpdates extends React.Component {
             ex => this.handleLoadError((ex.problem == "not-found") ? _("PackageKit is not installed") : ex));
     }
 
+    loadHistory() {
+        let history = [];
+
+        // would be nice to filter only for "update-packages" role, but can't here
+        pkTransaction("GetOldTransactions", [0], {
+                Transaction: (objPath, timeSpec, succeeded, role, duration, data) => {
+                    if (role !== PK_ROLE_ENUM_UPDATE_PACKAGES)
+                        return;
+                    // data looks like:
```

Comment:
I've never seen it having extra space. But an extra `trim()` doesn't hurt, I added it.